features/device-store/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Acknowledge, this code was belongs to IoT server ES extension app which is deprecated after moving to UUF based devicemgt app. So unable to fix the copyright year.